### ISSUE TITLE
Scope not detected inside function arguments

### ISSUE THREAD

**Issue by sirbrillig** [sirbrillig/phpcs-variable-analysis#13]
In the following example (derived from an [atoum](http://atoum.org/) test), `$service` is reported as being undefined.

```php
$this->given(
  $service = new Service(),
  $service->something()
);
```
